### PR TITLE
Issue 1120 fix iks deployment on ubi images

### PR DESCRIPTION
**Pull request by TiborBeres**
The base image of the VGW containers was switched from Ubuntu to RHEL UBI. This change broke the iks delivery scripts as the UBI base image does not contain the hostname utility. We have to supply a new delivery script that relies on python vs hostname.

### CODE REVIEW

**Comment by TiborBeres** [comment]
fixes #1120

File: kubernetes/bluemix/single-tenant/deploy_vgw_v102x.json [hunk 1]
```diff
@@ -0,0 +1,175 @@
+{
```
Comment (jfmartinez) [inline kubernetes/bluemix/single-tenant/deploy_vgw_v102x.json]: Can this name be renamed to be `kubernetes/bluemix/single-tenant/deploy.json`, this way its the default file users use when using the latest images.

Comment (TiborBeres) [reply kubernetes/bluemix/single-tenant/deploy_vgw_v102x.json]: so you would prefer this to be the default deploy.json and have the previous version of the file renamed deploy_vgw_v1.0.x,json?

Comment (TiborBeres) [reply kubernetes/bluemix/single-tenant/deploy_vgw_v102x.json]: it can be done of course if that's your preference

Comment (jfmartinez) [reply kubernetes/bluemix/single-tenant/deploy_vgw_v102x.json]: yup - it should be the new default deploy script

Comment (TiborBeres) [reply kubernetes/bluemix/single-tenant/deploy_vgw_v102x.json]: I have updated the multinode deployment scripts as well and I renamed the files

**Pull request merged**